operations/reservations.md

```diff
@@ -47,240 +44,246 @@ Returns all reservations within scope of the Access Token, filtered according to
 | `ChannelManagerNumber` | string | optional | Unique number of the reservation within the reservation group. |
-| `CancellationReason` | string [Cancellation reason](#reservation-cancellation-reason) | optional | Cancellation reason of the reservation. |
-| `ReleasedUtc` | string | optional | Date when the optional reservation is released in UTC timezone in ISO 8601 format. |
-| `Purpose` | string [Reservation purpose](#reservation-purpose) | optional | Purpose of the reservation. |
-| `Options` | string [Service order options](serviceorders.md#service-order-options) | optional | Options of the reservations. |
-| `QrCodeData` | string | optional | Reservation data for QR code generation. |
-| `PersonCounts` | array of [Person count](#person-count) | required | Number of people per age category the reservation was booked for. |
-
-#### Person count
-
-| Property | Type | Contract | Description |
-| :-- | :-- | :-- | :-- |
-| `AgeCategoryId` | string | required | Unique identifier of the [Age category](agecategories.md#age-category). |
-| `Count` | integer | required | Number of people of a given age category. Only positive value is accepted. |
+| `CancellationReason` | [Reservation cancellation reason](reservations.md#reservation-cancellation-reason) | optional | Cancellation reason of the reservation. |
+| `ReleasedUtc` | string | optional | Date when the optional reservation is released in UTC timezone. |
+| `ScheduledStartUtc` | string | required | Scheduled start time of reservation in UTC timezone. |
+| `ActualStartUtc` | string | optional | Actual customer check-in time of reservation in UTC timezone. |
+| `ScheduledEndUtc` | string | required | Scheduled end time of reservation in UTC timezone in ISO 8601 format. |
+| `ActualEndUtc` | string | optional | Actual end time of reservation in UTC timezone in ISO 8601 format. |
+| `Purpose` | [Reservation purpose](reservations.md#reservation-purpose) | optional |  |
+| `QrCodeData` | string | optional | QR code data of the reservation. |
+| `PersonCounts` | array of [Age category parameters](reservations.md#age-category-parameters) | required | Number of people per age category the reservation was booked for. At least one category with valid count must be provided. |
+| ~~`StartUtc`~~ | ~~string~~ | ~~required~~ | ~~Reservation start or check-in time (if it's earlier than scheduled start) in UTC timezone in ISO 8601 format.~~ **Deprecated!** |
+| ~~`EndUtc`~~ | ~~string~~ | ~~required~~ | ~~Scheduled end time of reservation in UTC timezone in ISO 8601 format~~ **Deprecated!** |
 
-#### Reservation state
+#### Service order state
 
-* `Enquired` - Confirmed neither by the customer nor enterprise.
-* `Optional` - Confirmed by enterprise but not by the guest \(the enterprise is holding resource for the guest\).
+* `Inquired` - Confirmed neither by the customer nor enterprise.
```

Comment:
Current page does not have the word "inquired" in it, only "enquired". Is this addition correct?

Reply:
@jonas-paul I have a note about this, see this Slack discussion -> https://mews.slack.com/archives/C01TCGUJ5FV/p1706089567735419